vignettes/opt-dead-store.Rmd

```diff
@@ -30,6 +30,10 @@ speed_up <- function(obj) {
 
 # Dead Store Elimination
 
+## Idea 
+
+The removal of a value allocated to a variable when it is not being read by any following instuction is called Dead Store Elimination. This Optimization is neccessary as it saves processor time and memory which ultimately results in the faster execution of program.
```

Comment:
`instuction` ~> `instruction`
`neccessary` ~> `necessary`
`execution of program` ~> `execution of the program`